src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/types';
+import classNames from 'classnames';
+
+type TodoInfoProps = {
+  todo: Todo;
+};
+
+export const TodoInfo: FC<TodoInfoProps> = ({ todo }) => {
+  return (
+    <article
+      className={classNames(
+        `TodoInfo ${todo.completed && 'TodoInfo--completed'}`,
+      )}
+      data-id={todo.id}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
```suggestion
     {todo.user && <UserInfo user={todo.user} />}
```
better to add the check here, so the UserInfo component can just render user info without any logic there